bundle-workflow/README.md

```diff
@@ -216,3 +275,24 @@ The following options are available.
 |--------------------|-------------------------------------------------------------------------|
 | --keep             | Do not delete the temporary working directory on both success or error. |
 | -v, --verbose      | Show more verbose output.                                               |
+
+### Component Onboarding
+
+1. Update a [manifest](/manifests) for a particular release to include your plugin.  For example to be included in the 1.1.0 release, you would update [opensearch-1.1.0.yml](/manifests/1.1.0/opensearch-1.1.0.yml). 
+   We require your plugin name, repository url, and git ref
+   that should be used. For unreleased versions this should be a branch in your repository.  Once a release is cut, these refs will be updated
+   to build from a tag or specific commit hash.
+
+2. Create a build.sh if you have specific requirements that are not covered by the [default build.sh script](/scripts/default/build.sh) and commit it to your repository. It
+   can live at the root of your repository or preferably scripts/build.sh to prevent noise at the root.
+
+3. Ensure your build.sh reads and passes along both `-Dbuild.snapshot` and `-Dopensearch.version` flags.  Snapshot builds should produce a
+   -SNAPSHOT tagged artifact for example `opensearch-plugin-1.1.0.0-SNAPSHOT.zip` where a release build of the same component would produce `opensearch-plugin-1.1.0.0.zip`.
+   The opensearch version flag ensures that each component is built against the same commit of the core repo.
```

Comment:
We don't call this a "core" repo. I would just remove this because this is a list of requirements.